deploy-service/common/src/main/java/com/pinterest/deployservice/bean/EnvironBean.java

```diff
@@ -72,10 +75,12 @@
     private String env_id;
 
     @NotEmpty
+    @Pattern(regexp="^[A-Za-z0-9_\\-]*$")
```

Comment:
can we do a DB review on the existing data to see if there is any violation? otherwise, some existing name might not work.

Reply:
yup, Jeremy and I discussed this in his original PR. I will double check.